src/configure/helper/repoAnalysisHelper.ts

```diff
@@ -28,7 +32,15 @@ export class RepoAnalysisHelper {
             repositoryAnalysisRequestBody.Repository = repositoryDetails;
             repositoryAnalysisRequestBody.WorkingDirectory = workspacePath;
 
-            repositoryAnalysisResponse = await this.portalExtensionClient.getRepositoryAnalysis(repositoryAnalysisRequestBody);
+            if (this.redirectHelper.repoAnalysisServiceFramework === ServiceFramework.Vssf) {
+                repositoryAnalysisRequestBody.Repository.authorizationInfo.scheme = "Token";
```

Comment:
Wouldn't we need these two lines for moda client call as well?

Reply:
otherwise you are not passing token for service to make authorized calls to github repo

Reply:
In case of Moda we're authenticating using github pat, so this can be avoided as it's redundant

Reply:
this is not about authneticating to moda based service. This is about template service using eight credential when making calls to github repo. For that it needs a credential parameter along with repo information.

Reply:
Yes I was thinking of using token from auth header, why'd you feel so? Are we changing it in template service as well?

Reply:
I mean, ideally calling moda service is authenticating to github, so we can use the same auth header for all internal calls as well